microsoft-edge/webview2/release-notes/archive.md

```diff
@@ -2353,7 +2353,7 @@ The following Experimental APIs have been added in this Prerelease SDK.
 * [ICoreWebView2Experimental21::remove_LaunchingExternalUriScheme](/microsoft-edge/webview2/reference/win32/icorewebview2experimental21?view=webview2-1.0.1724-prerelease&preserve-view=true#remove_launchingexternalurischeme)
 
 * [ICoreWebView2ExperimentalLaunchingExternalUriSchemeEventHandler](/microsoft-edge/webview2/reference/win32/icorewebview2experimentallaunchingexternalurischemeeventhandler?view=webview2-1.0.1724-prerelease&preserve-view=true)
-  * [ICoreWebView2ExperimentalLaunchingExternalUriSchemeEventHandler::Invoke](/microsoft-edge/webview2/reference/win32/icorewebview2experimentallaunchingexternalurischemeeventhandler?view=webview2-1.0.1724-prerelease&preserve-view=true#invoke)
+  * [ICoreWebView2ExperimentalLaunchingExternalUriSchemeEventHandler::Invoke](/microsoft-edge/webview2/reference/win32/icorewebview2experimentallaunchingexternalurischemeeventhandler?view=webview2-1.0.1724-prerelease&preserve-view=true#invoke)<!-- todo: delete? -->
```

Comment:
keep or delete this Invoke link?
rendered live article section: March 2023:
https://learn.microsoft.com/en-us/microsoft-edge/webview2/release-notes/archive?tabs=win32cpp#experimental-apis-7 
There IS an Invoke section there, so i plan to keep link and make a hidden note to keep it:
https://learn.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2experimentallaunchingexternalurischemeeventhandler?view=webview2-1.0.1724-prerelease&preserve-view=true#invoke
@peiche-jessica @victorhuangwq

Reply:
resolved: i appended hidden comment:
```
  * [ICoreWebView2ExperimentalLaunchingExternalUriSchemeEventHandler::Invoke]
(/microsoft-edge/webview2/reference/win32/
icorewebview2experimentallaunchingexternalurischemeeventhandler
?view=webview2-1.0.1724-prerelease
&preserve-view=true#invoke)<!-- keep this Invoke, b/c listed in api ref -->
```
since this particular Invoke is listed in API Ref, it's consistent to continue listing it in RelNotes.
@peiche-jessica @victorhuangwq